sdk/search/search-documents/src/searchIndexClient.ts

```diff
@@ -552,16 +554,24 @@ export class SearchIndexClient {
   /**
    * Calls an analyzer or tokenizer manually on provided text.
    * @param indexName The name of the index that contains the field to analyze
+   * @param text The text to break into tokens.
    * @param options Additional arguments
    */
-  public async analyzeText(indexName: string, options: AnalyzeTextOptions): Promise<AnalyzeResult> {
+  public async analyzeText(
+    indexName: string,
+    text: string,
+    options: AnalyzeTextOptions
+  ): Promise<AnalyzeResult> {
     const { operationOptions, restOptions } = utils.extractOperationOptions(options);
 
     const { span, updatedOptions } = createSpan("SearchIndexClient-analyzeText", operationOptions);
     try {
       const result = await this.client.indexes.analyze(
         indexName,
-        restOptions,
+        {
+          ...restOptions,
+          text: text
```

Comment:
style nit: can just write `text` here instead of duplicating the name

Reply:
Done